docs/how-to-guides/SPICE/creating-spice-config-files-in-spiceql.md

```diff
@@ -0,0 +1,253 @@
+# Creating New Mission Config Files
+
+## Required Reading
+
+There are many niche topics important to understand how SPICE and kernels work. The important things to know: 
+
+1. What are kernels?
+2. What are the different kernel types?
+3. What are the different kernel qualities?
+
+Here are some resources: 
+
+* [Intro to Kernels](https://naif.jpl.nasa.gov/pub/naif/toolkit_docs/Tutorials/pdf/individual_docs/12_intro_to_kernels), includes descriptions of kernel types:
+* [Useful glossary of terms](../../concepts/glossary/glossary.md)
```

Comment:
This is not totally related to the SPICE-specific PR but I noticed that the glossary endpoint looks like `http://127.0.0.1:8000/concepts/glossary/glossary/`. Maybe we can move `glossary.md` out of the `docs/concepts/glossary` folder and remove the `glossary/` folder unless there may be more items under `glossary/` in the future.

Reply:
hm I was able to move the glossary.md file up a level and remove the `glossary/` folder, then changed mkdocs.yml to point `Glossary` to `concepts/glossary.md` and that rendered the glossary page to this endpoint: http://127.0.0.1:8000/concepts/glossary/

Reply:
probably a skill issue on my end, I'll make that change.